pkg/credentialprovider/aws/aws_credentials.go

```diff
@@ -85,12 +148,26 @@ func (p *ecrProvider) Enabled() bool {
 func (p *ecrProvider) Provide(image string) credentialprovider.DockerConfig {
 	parsed, err := parseRepoURL(image)
 	if err != nil {
-		klog.V(3).Info(err)
+		klog.V(5).Info(err)
```

Comment:
this should not log anything if the regex simply didn't match

Reply:
mentioned above as:

> drop the klog.V(3).Info(err) output if a non-ECR image is passed to provide... this isn't something we need to log

Reply:
Yeah I reduced to V5 because I found it useful for testing but happy to remove it.

Reply:
Removed in latest push.